m_etcd/coordinator.go

```diff
@@ -480,13 +480,20 @@ func (ec *EtcdCoordinator) watch(path string, index uint64, stop chan bool) (*et
 	const recursive = true
 	for {
 		// Start the blocking watch after the last response's index.
-		rawResp, err := ec.Client.RawWatch(path, index+1, recursive, nil, stop)
+		rawResp, err := protectedRawWatch(ec.Client, path, index+1, recursive, nil, stop)
 		if err != nil {
 			if err == etcd.ErrWatchStoppedByUser {
 				// This isn't actually an error, the stop chan was closed. Time to stop!
 				return nil, err
 			}
 
+			// This is probably a canceled request panic
+			// Wait a little bit, then continue as normal
```

Comment:
Mind mentioning "remove post-Go-1.5" or something in the comment? I know you documented it in protectedRawWatch, but for workarounds like this I like to make it obvious when we can kill it.